scalameta/contrib/src/main/scala/scala/meta/contrib/Equal.scala

```diff
@@ -0,0 +1,12 @@
+package scala.meta.contrib
+
+import scala.meta.prettyprinters.Show
```

Comment:
It would seem that `Show` is now used for multiple heterogeneous purposes. How about we move it to a different package?